src/index.js

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable no-loop-func */
```

Comment:
**You did a great job, all the functions are working as expected.**

- [ ] However, we got one issue that needs to be fixed. In this project, students are not allowed to use eslint disabler. 
Kindly fix the issues and remove the eslint disabler.

Reply:
@Rustamxon7 Please help me since I do not know how to fix this problem

Reply:
Hello, @abenaTenkorang Sorry for the late reply...

Reply:
At that point. You have used no-loop-func disabling that is, you shouldn't declare a function inside the loop. I suggest you create all declared functions outside of the loop and then you can reuse them inside of the loop again.

Reply:
Issue resolved. Thank you @Rustamxon7